deploy/aks/monitoring/perf/README.md

```diff
@@ -0,0 +1,112 @@
+## Performance Test Dashboard Setup
+
+This document serves as a guideline for setting up a performance test dashboard using Azure managed Prometheus and Grafana. We will be creating an AKS (Azure Kubernetes Service) cluster and installing Prometheus Pushgateway and Azure Monitoring Agent (AMA) to scrape performance test metrics and push them to Azure managed Prometheus. Additionally, we will set up an ingress controller with authentication to act as a proxy for Prometheus Pushgateway. Finally, we will create a Grafana dashboard by importing a predefined JSON model.
+
+Below are step-by-step instructions to set up a performance test dashboard using Azure managed Prometheus and Grafana. Follow these steps to configure your performance monitoring environment.
+
+#### Step 1: Clone the test-infra repo
+
+```bash
+git clone https://github.com/dapr/test-infra.git
+cd deploy/aks/monitoring/perf
+```
+
+#### Step 2: Login to Azure and set your subscription
+
+```bash
+az login
+export SUBSCRIPTION_ID=<SUBSCRIPTION UUID TO BE USED FOR THE COMMANDS BELLOW>
+az account set --subscription "${SUBSCRIPTION_ID}"
+```
+
+### Step 3: Set resource group name, location, cluster name and use the same namespace (dapr-perf-metrics)
+
+```bash
+DAPR_PERF_RG=<resource group to be used>  
+DAPR_PERF_LOCATION=<insert region>  
+CLUSTER_NAME=<cluster name>
+DAPR_PERF_METRICS_NAMESPACE=dapr-perf-metrics
+```
+
+#### Step 4: Create Resource Group
+```bash
+az group create --name $DAPR_PERF_RG --location $DAPR_PERF_LOCATION
+```
+
+#### Step 5: Execute main.bicep and provide AKS cluster name on prompt
+
+```bash
+az deployment group create --resource-group $DAPR_PERF_RG --template-file main.bicep --parameters clusterName="${CLUSTER_NAME}"
+```
+
+#### Step 6: Merge Newly Created Cluster Username and Password
+
+```bash
+az aks get-credentials --resource-group $DAPR_PERF_RG --name "${CLUSTER_NAME}"
+```
+
+#### Step 7: Switch AKS Cluster Context
+
+```bash
+kubectl config use-context "${CLUSTER_NAME}"
+```
+
+#### Step 8: Install Prometheus Pushgateway
+
+```bash
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm repo update
+helm install --namespace $DAPR_PERF_METRICS_NAMESPACE prometheus-pushgateway prometheus-community/prometheus-pushgateway
```

Comment:
I don't think you created this namespace before.  How about letting helm take care of this. And how about making this command idempotent? 

```suggestion
helm upgrade --install \
    prometheus-pushgateway prometheus-community/prometheus-pushgateway \
    --namespace $DAPR_PERF_METRICS_NAMESPACE \
    --create-namespace \
    --wait
```